pkgs/servers/mastodon/update.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, writeScript, fetchFromGitHub, nix-prefetch-github, yarn2nix-moretea, bundix, coreutils, jq, curl, nix, git, gawk, gnused }:
+
+let
+  # until submodules support gets merged
+  bundix_with_submodules = bundix.overrideAttrs (attrs: {
+    src = fetchFromGitHub {
+      owner = "petabyteboy";
+      repo = "bundix";
+      rev = "feature/submodules";
```

Comment:
Should be pinned to a SHA (or, preferably, just patch upstream Bundix with `fetchpatch`). As it stands, this is not reproducible.

Reply:
This will be gone very soon anyways

Reply:
gone